### PR TITLE
Fixes/space removed between dc member and keyword

### PR DESCRIPTION
**Pull request by KalyanaChakravarthi**
Fixes issues 1249 and 1005. 1288 issue is caused by the previous fix for 1005.
Corrected the fix for 1005, so that issue 1288 will be resolved.

### CODE REVIEW

File: src/chunk_list.h [hunk 1]
```diff
@@ -797,6 +798,16 @@ static_inline bool chunk_is_forin(chunk_t *pc)
...
 
 
+/**
+ * Checks if a chunk is a keyword
+ *
+ * @return true  - the chunk is keyword
+ */
+static_inline bool chunk_is_keyword(chunk_t *pc)
+{
+   return(find_keyword_type(pc->text(), strlen(pc->text())) != CT_WORD);
```
Comment (hschlichter) [inline src/chunk_list.h:808]: Small stylistic comment. You shouldn't wrap it in parens unless it has semantic value. Which it doesn't do in this case.

When quickly scanning the code, you would think that return is a function.

**Review by hschlichter — CHANGES_REQUESTED** [review]
Besides the stylistic comment, it looks good. But please fix :-)

File: src/chunk_list.h (hunk 1 above)
Comment (mihaipopescu) [reply src/chunk_list.h:808]: Unfortunately it's the style chosen and enforced by upstream.

Comment (hschlichter) [reply src/chunk_list.h:808]: Hmm ... ok. Makes me a bit worried that it actually adds the parens. But if this is the style for the uncrustify repo, then thats how it is.

**Review by hschlichter — APPROVED** [review]

**Pull request merged**